resources/maskinporten_clients.py

```diff
@@ -171,20 +165,23 @@ def create_client_key(
     )
 
     key_password = generate_password(pw_length=32)
-
-    try:
-        send_secrets_service.send_secrets(
-            secrets=Secrets(
-                keystore=pkcs12_from_key(key, key_password),
-                key_id=key_id,
-                key_password=key_password,
-            ),
-            maskinporten_client_id=client_id,
-            destination_aws_account_id=body.destination_aws_account,
-            destination_aws_region=body.destination_aws_region,
-        )
-    except AssumeRoleAccessDeniedException as e:
-        raise ErrorResponse(status.HTTP_422_UNPROCESSABLE_ENTITY, str(e))
+    keystore = pkcs12_from_key(key, key_password)
+    ssm_params = None
+
+    if body.destination_aws_account:
```

Comment:
Sjekke om region også er satt, `send_to_aws = account and region`? Vet ikke hva send_secrets med region=None ?